### PR TITLE
new: issue templates

### PR DESCRIPTION
**Pull request by neuronflow**
Signed-off-by: neuronflow <florian.kofler@tum.de>

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: '[BUG] '
+labels: ''
```
Comment (MarcelRosier) [inline .github/ISSUE_TEMPLATE/bug_report.md:5]: maybe add bug label?

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: '[FEATURE] '
+labels: ''
```
Comment (MarcelRosier) [inline .github/ISSUE_TEMPLATE/feature_request.md:5]: feature label

File: .github/ISSUE_TEMPLATE/question.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+---
+name: Question
+about: Please ask your question, make sure to read the FAQ before
+title: '[QUESTION] '
+labels: ''
```
Comment (MarcelRosier) [inline .github/ISSUE_TEMPLATE/question.md:5]: question label

**Review by MarcelRosier — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/question.md (hunk 3 above)
Comment (neuronflow) [reply .github/ISSUE_TEMPLATE/question.md]: I hope I did it correctly

**Review by MarcelRosier — APPROVED** [review]

**Pull request merged**